variables.mk

```diff
@@ -170,6 +170,7 @@ SBT_NON_THIN ?= $(subst $(SBT_CLIENT_FLAG),,$(SBT))
 define run_scala_main
 	cd $(base_dir) && $(SBT) ";project $(1); runMain $(2) $(3)"
 endef
+run_main = cd $(base_dir) && java $(JAVA_ARGS) -cp $(FAT_JAR) $(2) $(3)
```

Comment:
I assume you mean `$(JAVA_OPTS)` here? This API changed since you last worked on this (I also see this change needing to be done in the FireSim PR).

Additionally, I think we should match these functions (either both have "define" or both are one-liners).

Finally, this is really just used in FireSim correct (in `target-agnostic.mk`), I wonder if this makes more sense to live in FireSim instead of Chipyard's makefiles especially since Chipyard doesn't use this.

Reply:
took this back out of variables.mk since FireSim only uses it in 
090eb1a 
I also addressed the `JAVA_ARGS` uses in firesim.  Thanks for catching this!